spec/models/form_profile_spec.rb

```diff
@@ -9,7 +9,7 @@
   let(:user) { build(:user, :loa3) }
 
   before do
-    user.va_profile.suffix = 'Jr.'
+    user.mpi.profile.suffix = 'Jr.'
```

Comment:
If we could prevent this case of referring to `user.mpi`, that'd help us down the line, you can do that by adding:
`  let(:mvi_profile) { build(:mvi_profile, suffix: 'Jr.') }`
    
`before {
      stub_mpi(mvi_profile)
    }`

Reply:
I tried replacing this case but it broken a few dozen other tests in this file with a variety of different user profile attributes being stubbed incorrectly, my guess is due to the unique nature of it testing forms?